### DISCUSSION TITLE
Configuracion de Octodash URL

### DISCUSSION THREAD

**Discussion by javierzg** [UnchartedBull/OctoDash#3246] [Q&A] [answered]
Cheers
I have the following serious problem, my son thinking that he was doing a good thing, he mistakenly changed the URL in the octodash configuration, changing the localHost for a URL number that in no case is the address given by Pi.
I enclose a photo where you can see Initializing takes two hours like this without being able to connect.
Please could you guide me to solve this problem.

Thank you

![20221015_230244](https://user-images.githubusercontent.com/74110402/196014860-57826422-34dd-46fd-b659-7c006e25255c.jpg)

**Comment by jneilliii** [comment, marked as answer]
Sometimes the easiest way out of that is deleting the config.json file and restarting OctoDash and setup from scratch. I would definitely use the manual option and http://localhost:5000/ or http://127.0.01:5000 for the URL for best performance.

**Reply by javierzg** [reply]
Hello
jneilliii
Thanks for your help, how could I delete what commands do I need to delete config.json
and forgive my questions.

**Reply by jneilliii** [reply]
you have to SSH to the pi or login local and then run these commands

```
rm ~/.config/octodash/config.json
sudo service getty@tty1 restart
```

**Reply by jneilliii** [reply]
actually, you could just install my OctoDash Companion plugin in OctoPrint and edit the URL back to the localhost address from OctoPrint's settings and then use the system menu items it add to stop OctoDash and start OctoDash.

**Comment by javierzg** [comment]
Thank you very much for your advice and your time, problem solved
